packages/medusa/package.json

```diff
@@ -133,9 +133,13 @@
     "@mikro-orm/migrations": "6.4.3",
     "@mikro-orm/postgresql": "6.4.3",
     "awilix": "^8.0.1",
+    "react-dom": "^18.0.0",
     "yalc": "1.0.0-pre.53"
   },
   "peerDependenciesMeta": {
+    "react-dom": {
+      "optional": true
+    },
```

Comment:
`admin-bundler` has it as a peer dependency and `@medusajs/medusa` install `admin-bundler`. Therefore we have to bubble up all the peer dependencies.